app/graphql/resolvers/event_records_search.rb

```diff
@@ -11,15 +11,15 @@ class Resolvers::EventRecordsSearch
 
     begin
       lookahead = context[:extras][:lookahead].selection(:event_records)
+      event_records = event_records.upcoming_with_date_select if lookahead.selects?(:date)
       event_records = event_records.includes(:addresses) if lookahead.selects?(:addresses)
       event_records = event_records.includes(:categories) if lookahead.selects?(:categories)
-      event_records = event_records.includes(:dates) if lookahead.selects?(:list_date) || lookahead.selects?(:dates)
       event_records = event_records.includes(:data_provider) if lookahead.selects?(:data_provider)
     rescue
       # ignore
     end
 
-    event_records.distinct
+    event_records
```

Comment:
warum ist das besser so ohne distinct?

Reply:
we want multiple objects for same events, so this cannot be distinct anymore, which would result in just the amount of objects like unique events.

Reply:
wow... ok....

Reply:
and in my checks there were no differences in performance for the "old way" of request

Reply:
not in performance, but in getting duplicate Events ? in some cases... , maybe?...

Reply:
wasn't the case. but i will have an eye on this again when testing before merge.

Reply:
I could not find a bad case, but will ensure the existing behavior with:

```
if lookahead.selects?(:date)
  event_records
else
  event_records.distinct
end
```